css/style.css

```diff
@@ -16,4 +22,125 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* Navigation styles */
+.meet-karma {
+  font-weight: bold;
+}
+
+.nav-bar {
+  background: #ffffff;
+  overflow: hidden;
+  position: fixed;
+  width: 100%;
+  top: 0;
+}
+
+.logo {
+  float: left;
+  height: 50px;
+  width: 50px;
+  padding: 10px 0 10px 100px;
+}
+
+nav {
+  float: right;
+}
+
+nav ul {
+  margin: 0;
+  padding-right: 100px;
+  list-style: none;
+}
+
+nav li {
+  display: inline-block;
+  margin-left: 40px;
+  padding-top: 30px;
+}
+
+nav a {
+  color: #444242;
+  text-decoration: none;
+}
+
+nav a:hover {
+  color: #ffa500;
+}
+
+/* section 1 */
+h1 {
+  font-size: 80px;
+  font-weight: lighter;
+}
+
+.ptag {
```

Comment:
it is better to name your class in a descriptive way not ptag as it is not a correct way of naming classes